archapp/interactive.py

```diff
@@ -101,13 +102,24 @@ def get(self, pvname=None, start=30, end=None, unit="days", chunk=False, xarray=
         data : np.ndarray or xarray
         """
         if pvname is None:
-            raise NotImplementedError()
+            pvname = self._last_pvname
+            if pvname is None:
+                raise ValueError('No cached pvname. Must provide the '
+                                 'first pvname of the session.')
+        else:
+            self._last_pvname = pvname
+
         pvs = self._expand_pvnames(pvname)
         dt_objs = sorted(interactive_args(start, end, unit))
         xarr = self._data.get(pvs, dt_objs[0], dt_objs[1], chunk=chunk)
         if xarray:
             return xarr
-        raise NotImplementedError()
+        # Unpack the xarray as a np.ndarray of just the values
+        values = [np.datetime_as_string(xarr.time.values)]
+        for var in xarr.variables:
+            if var not in ('time', 'field'):
+                values.append(xarr[var].values[0])
+        return np.column_stack(values)
```

Comment:
My only concern is that the field/PV names get lost during this process. If searching for multiple PVs, those could be helpful. Maybe something like below (with `headers` defaulting to `False`)? There's got to be a better way to grab the field names and add them but I can't figure it out tonight.

```suggestion
        # Unpack the xarray as a np.ndarray of just the values
        values = [np.datetime_as_string(xarr.time.values)]
        for var in xarr.variables:
            if var not in ('time', 'field'):
                values.append(xarr[var].values[0])
        arr = np.column_stack(values)
        if headers:
            np.insert(arr, 0, [key for key in xarr.variables if key is not 'field'], axis=0)
        return arr
```

Reply:
Let's look at doing this in a follow-up PR